json/pt.json

```diff
@@ -8,15 +8,15 @@
     "E-Mail Address": "Endereço de e-mail",
     "Forbidden": "Proibido",
     "Forgot Your Password?": "Esqueceu-se da Palavra-passe?",
-    "Go Home": "Voltar para o início",
+    "Go Home": "Voltar ao início",
     "Hello!": "Olá!",
     "If you did not create an account, no further action is required.": "Se não criou uma conta, ignore este e-mail.",
     "If you did not receive the email": "Se não recebeu o e-mail",
-    "If you did not request a password reset, no further action is required.": "Se não pediu para alterar a palavra-passe, ignore este e-mail.",
-    "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Se não conseguir clicar no botão \":actionText\", copie e cole o URL abaixo\nno seu browser:",
+    "If you did not request a password reset, no further action is required.": "Se não pediu para redefinir a palavra-passe, ignore este e-mail.",
+    "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Se não conseguir clicar no botão \":actionText\", copie e cole a URL abaixo\nno seu browser:",
     "Invalid signature.": "Assinatura inválida.",
-    "Login": "Entrar",
-    "Logout": "Sair",
+    "Login": "Iniciar Sessão",
```

Comment:
I have the same problem than Spanish version ( #1292 ). I prefer "Entrar" and "Sair", because it's short, and it's homogeneous with others languages

Reply:
I went with the Microsoft way I guess.

I don't mind "Entrar" but "Sair" I don't think it correctly reflects what your doing.

Reply:
@gh-rboliveira @WhereIsLucas What is your opinion about this translation ? thanks by advance

Reply:
In my opinion:
Login should be "Iniciar Sessão" and Logout "Terminar Sessão"
Entrar means enter, Sair means exit.

Reply:
Second that, cheers.

Reply:
I'm sorry I don't speak PT, I can't help here.

Reply:
> I'm sorry I don't speak PT, I can't help here.

sorry ;)

Reply:
ok, I merge, so I must re open the pull request for Spanish too....

Reply:
Thank you man